app/src/main/java/org/mozilla/focus/widget/ResizableKeyboardViewDelegate.java

```diff
@@ -84,7 +76,6 @@ public void onGlobalLayout() {
                 0, 0);
 
         try {
-            idOfViewToHide = styleAttributeArray.getResourceId(R.styleable.ResizableKeyboardViewDelegate_viewToHideWhenActivated, -1);
```

Comment:
I think we could use a typed Array here and pass the list of IDs through the XML (since we already have the functionality, and we just need to allow it to work for multiple views. What do you think?

```java
arrayOfViewsToHide = styleAttributeArray.getTextArray(R.styleable.ResizableKeyboardViewDelegate_viewToHideWhenActivated, -1);

// ...

if (arrayOfViewsToHide != null && !arrayOfViewsToHide.isEmpty()) {
    // loop through array and setVisibility
}
```

Then in an `arrays.xml`:

```xml
<?xml version="1.0" encoding="utf-8"?>
<resources>
    <array name="viewsToHide">
        <item>@id/erase</item>
        <item>@id/tabs</item>
    </array>
</resources>
```

The final usage in the layout file:

```xml
<org.mozilla.focus.widget.ResizableKeyboardCoordinatorLayout
    app:viewToHideWhenActivated="@array/viewsToHide"
    android:layout_marginTop="0dp"
    android:id="@+id/main_content"
    android:layout_width="match_parent"
```